docs/source/metadata.rst

```diff
@@ -66,6 +66,10 @@ your provider does not use the visitor pattern for computing metadata for a tree
 ------------------
 Metadata Providers
 ------------------
+:class:`~libcst.metadata.BasicPositionProvider`, :class:`~libcst.metadata.SyntacticPositionProvider`,
+:class:`~libcst.metadata.ExpressionContextProvider` and :class:`~libcst.metadata.ScopeProvider`
+are currently provided. Each metadata provider may has its own custom data structure. Read the following
```

Comment:
"Read the following document for more detail." seems like an extraneous sentence because the user is already reading the document.